kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -452,15 +455,13 @@ static Condition resolvedRefsUnknown(Clock clock,
         return newUnknownCondition(clock, observedGenerationSource, Condition.Type.ResolvedRefs, e);
     }
 
-    static String slug(String singular, String group, String name) {
-        return singular + "." + group + "/" + name;
+    public static String slug(String singular, String group, String name) {
```

Comment:
If we really want to include namespace in all the places then we should make this `private`

Reply:
done